03week/ticTacToe.js

```diff
@@ -24,24 +24,57 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  if ((playerTurn == board[0][0] && playerTurn == board[0][1] && playerTurn == board[0][2]) 
+  || (playerTurn == board[1][0] && playerTurn == board[1][1] && playerTurn == board[1][2]) 
+    || (playerTurn == board[2][0] && playerTurn == board[2][1] && playerTurn == board[2][2])) {
+    console.log(playerTurn + " wins!");
+    return true;
+  }
 }
 
 function verticalWin() {
-  // Your code here
+  if ((playerTurn == board[0][0] && playerTurn == board[1][0] && playerTurn == board[2][0]) 
+  || (playerTurn == board[0][1] && playerTurn == board[1][1] && playerTurn == board[2][1]) 
+    || (playerTurn == board[0][2] && playerTurn == board[1][2] && playerTurn == board[2][2])) {
+    console.log(playerTurn + " wins!");
+    return true;
+  }
 }
 
 function diagonalWin() {
-  // Your code here
+  if ((playerTurn == board[0][0] && playerTurn == board[1][1] && playerTurn == board[2][2]) 
+  || (playerTurn == board[2][0] && playerTurn == board[1][1] && playerTurn == board[0][2])) {
+  console.log(playerTurn + " wins!");
+  return true; 
+  }
 }
 
 function checkForWin() {
-  // Your code here
+ horizontalWin()
+  verticalWin()
+  diagonalWin()
+
+  if (horizontalWin || verticalWin || diagonalWin){
+  return true;
+}
 }
 
```

Comment:
looks a bit incomplete here.

Reply:
Hey Dinh! I think you mentioned last night that you don't close the merge requests until we address your comments. I assume that may be why I haven't received a grade on this as well? All of the tests ran properly for tic tac toe with this code, so would you please give me more clarity on what I'm missing or what you need to complete this assignment.